### PR TITLE
Replace hardcoded docs url with a constant.

### LINKED ISSUE #1200: [mesheryctl] Create global constant to reference Meshery docs site
**Current Behavior**
`mesheryctl` has the URL to the Meshery documentation site hard-coded in some areas. Example:

```
$ mesheryctl perf
Error: please enter a test URL
See https://meshery.layer5.io/docs/guides/mesheryctl#performance-management for usage details
```

**Desired Behavior**
- [ ] A `DOCS_BASEURL` should be created as a global reference in `mesheryctl`.
- [ ] Each of the hard-coded references to Meshery documentation should be updated with this global constant.

### PR DESCRIPTION
**Pull request by Sylfrena**
Signed-off-by: Sumera Priyadarsini <sylphrenadin@gmail.com>

**Description**

This PR fixes #1200

**Notes for Reviewers**
I wasn't able to run the cli tool after making my changes, there seems to be something wrong with my local setup, so if someone could pull this branch and run it locally before merging it would be a great help

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by kushthedude — CHANGES_REQUESTED** [review]
Please fix the golangci errors

File: mesheryctl/pkg/utils/helpers.go [hunk 1]
```diff
@@ -31,10 +31,12 @@ const (
 	dockerComposeBinary         = "/usr/local/bin/docker-compose"
 
 	// Usage URLs
-	rootUsageURL   = "https://meshery.layer5.io/docs/guides/mesheryctl#global-commands-and-flags"
-	perfUsageURL   = "https://meshery.layer5.io/docs/guides/mesheryctl#performance-management"
-	systemUsageURL = "https://meshery.layer5.io/docs/guides/mesheryctl#meshery-lifecycle-management"
-	meshUsageURL   = "https://meshery.layer5.io/docs/guides/mesheryctl#service-mesh-lifecycle-management"
+	DOCS_BASEURL = "https://meshery.layer5.io/docs/guides/mesheryctl"
```
Comment (kushthedude) [inline mesheryctl/pkg/utils/helpers.go:34]: Only put docs base URL in constants & please follow camel case

Comment (leecalcote) [reply mesheryctl/pkg/utils/helpers.go:34]: An example being ```docsBaseURL = "https://meshery.layer5.io/docs/"```

Right, @kushthedude?

**Comment by leecalcote** [comment]
Very pleased to see this, @Sylfrena! Thank you. 😄

**Comment by leecalcote** [comment]
@Sylfrena one of the lint checks failed here. I wonder if you can take a look and correct - https://github.com/layer5io/meshery/pull/1496/checks?check_run_id=1048206528#step:3:61

**Comment by leecalcote** [comment]
@Sylfrena You're right. When I checkout this branch and build from the `/mesheryctl/` directory, here is what is returned (similar to the lint check errors):

```
 make
go build -ldflags="-w -s -X github.com/layer5io/meshery/mesheryctl/internal/cli/root.version=v0.4.4 -X github.com/layer5io/meshery/mesheryctl/internal/cli/root.commitsha=bf05e5735825ab931fe09c1decb53e278cccfdd4" -o mesheryctl cmd/mesheryctl/main.go
# github.com/layer5io/meshery/mesheryctl/pkg/utils
pkg/utils/helpers.go:36:2: const initializer fmt.Sprintf("%s#%s", DOCS_BASEURL, "global-commands-and-flags") is not a constant
pkg/utils/helpers.go:37:2: const initializer fmt.Sprintf("%s#%s", DOCS_BASEURL, "performance-management") is not a constant
pkg/utils/helpers.go:38:2: const initializer fmt.Sprintf("%s#%s", DOCS_BASEURL, "meshery-lifecycle-management") is not a constant
pkg/utils/helpers.go:39:2: const initializer fmt.Sprintf("%s#%s", DOCS_BASEURL, "service-mesh-lifecycle-management") is not a constant
make: *** [make] Error 2
```

**Comment by leecalcote** [comment]
@Sylfrena please ping @anirudhjain75 or @anandrkskd in Slack. Either would be happy to help here. 

It's very nice to have you contributing! 👍

**Review by leecalcote — APPROVED** [review]
Thank you for working on this issue, @Sylfrena 😄

**Comment by Sylfrena** [comment]
Hi @leecalcote,

I am so sorry I was unable to respond to this. My system broke, and then I fell sick so it took a while to get my entire setup environment in order. However, I was able to fix the error I was facing earlier- turns out I had not built the ui correctly, so the error was popping up. Thank you so much for taking this up and merging it, and I hope to be able to contribute better in the near future. :)

**Pull request merged**